integration/ports.go

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
can't we backport it without dragging this file along?

Reply:
Thanks Yacov. The file is removed.